packages/dappmanager/src/params.ts

```diff
@@ -30,6 +34,30 @@ if (process.env.TEST) {
 /** Absolute global ENVs .env file from DAPPMANAGER containers */
 const GLOBAL_ENVS_PATH = path.join(DNCORE_DIR, GLOBAL_ENVS_FILE_NAME);
 
+//Lodestar minimum versions
+const lodestarStakersMinimumVersions: {
+  [key: DnpName]: { [key: DnpName]: string };
```

Comment:
You can take advantage from the existing types for execution and consensus client types depending on the network:
```
ExecutionClient<Network>
ConsensusClient<Network>
```